deploy/kwok/setup.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/sh
+
+set -e
+
+: ${TMP_DIR:="/tmp"}
+: ${KWOK_WORKDIR:="$TMP_DIR/kwok"}
+export KWOK_WORKDIR
+
+: ${KUBECONFIG:="$TMP_DIR/kubeconfig.yaml"}
+export KUBECONFIG
+
+: ${PROVISIONER_IMAGE:="ghcr.io/adriansuarez/noop-provisioner:latest"}
```

Comment:
What is this?

Reply:
This is needed because there is no volume provisioner in KWOK, so pods that have PVCs never get scheduled because the PVCs remain indefinitely in a Pending state.

See https://github.com/kubernetes-sigs/kwok/issues/982 and https://github.com/adriansuarez/noop-provisioner for context.

Reply:
Thanks for explanation! Can you include a link to the repository somewhere in the script?